h5py/h5t.pyx

```diff
@@ -957,13 +960,22 @@ cdef class TypeFloatID(TypeAtomicID):
         # Handle non-standard exponent and mantissa sizes.
         for size, finfo in sorted(available_ftypes.items()):
             nmant = finfo.nmant
-            if nmant == 63 and finfo.nexp == 15:
+            maxexp = finfo.maxexp
+            minexp = finfo.minexp
+            # workaround for numpy's buggy finfo on float128 on ppc64 archs
+            if size == 16 and MACHINE.startswith('ppc64'):
```

Comment:
IIUC, this IBM double double is a regular double for sign and exponent, and the second double is used only for precision (mantissa).
So the exponent size is 11 bits, bias is 1023, and mandissa is 52 + 64 = 116.